iroh-net/src/net/network.rs

```diff
@@ -0,0 +1,46 @@
+use std::net::{IpAddr, Ipv4Addr, Ipv6Addr};
+
+/// Network selection between Ipv4 and Ipv6.
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub enum Network {
+    /// Ipv4
+    Ipv4,
+    /// Ipv6
+    Ipv6,
+}
+
+impl From<IpAddr> for Network {
+    fn from(value: IpAddr) -> Self {
+        match value {
+            IpAddr::V4(_) => Self::Ipv4,
+            IpAddr::V6(_) => Self::Ipv6,
+        }
+    }
+}
+
+impl Network {
+    /// Returns the matching default address.
+    pub fn default_addr(&self) -> IpAddr {
```

Comment:
should this not be better called `unspecified_addr`?

Reply:
fixed